fuel/schemes.py

```diff
@@ -232,3 +232,48 @@ def get_request_iterator(self):
         indices = list(self.indices)
         self.rng.shuffle(indices)
         return iter_(indices)
+
+
+def k_fold_cross_validation(scheme_class, num_examples, k, strict=True,
```

Comment:
I don't know if we need "k_fold" in the name; maybe just call it "cross_validation" and change "k" to "folds" or "num_folds"?